src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -28,6 +28,15 @@ define(['app-config', 'config', 'sulucontent/components/content/preview/main'],
             };
         },
 
+        collaboration: function() {
+            return {
+                autoStart: false,
```

Comment:
at least in the content form you have started the component quite late in the first version. i have tested it also without autoStart but the component was remove right after creating it. therefor i have introduced this flag for the content form.